scene/gui/code_edit.cpp

```diff
@@ -1498,9 +1498,9 @@ void CodeEdit::_line_number_draw_callback(int p_line, int p_gutter, const Rect2
 	ofs.y += TS->shaped_text_get_ascent(text_rid);
 
 	if (rtl) {
-		ofs.x = p_region.position.x;
-	} else {
 		ofs.x = p_region.get_end().x - text_size.width;
+	} else {
+		ofs.x = p_region.position.x;
 	}
```

Comment:
Change looks fine.

But I'm not sure why there's RTL case here, East Arabic numerals (١٢٣) are written in the same direction as West Arabic (123), and `CodeEdit` is always in LTR mode.

Reply:
for some reason CodeEdit actually can be in rtl mode? not normally, but it can be forced. and the gutters would be on the right side of the screen. theres cases for it all over in TextEdit, it just makes the spacing correct if it ever is in rtl mode :confused:
i could just take it out